src/index.html

```diff
@@ -4,9 +4,358 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="moyo-icon" href="#">
+        <img src="images/logo.png" alt="moyo_icon">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="cards">
+      <article data-qa="card" class="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="imac.jpeg"
+        >
+
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="logicBlock logicBlock--rating">
+          <span class="logicBlock__stars logicBlock__stars--4">
+            <span class="logicBlock__star"></span>
+            <span class="logicBlock__star"></span>
+            <span class="logicBlock__star"></span>
+            <span class="logicBlock__star"></span>
+            <span class="logicBlock__star"></span>
+          </span>
+
+          <p class="logicBlock__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="logicBlock logicBlock--price">
+          <p class="logicBlock__priceElement logicBlock__priceElement--text">Price:</p>
+          <p class="logicBlock__priceElement logicBlock__priceElement--value">$2,199</p>
+        </div>
+
+        <div class="button">
+          <a
+            data-qa="card-hover"
+            class="buttonText"
+            href="#"
+          >
+            buy
+          </a>
+        </div>
+      </article>
+      <article class="card">
```

Comment:
Add a blank line between the cards
```suggestion
      </article>
      
      <article class="card">
```